chained-actions/intro.md

```diff
@@ -0,0 +1,12 @@
+# Apache OpenWhisk on OpenShift
+
+Apache OpenWhisk allows chaining of actions.  We will now create a simple sequence of actions which will split, convert to uppercase, and
+ sort a comma separated string.  To highlight the flexibility of OpenWhisk, we will implement each step in a different language and 
+ seamless chain them together to construct the overall process.
+
```

Comment:
Add the following for now:

**Currently, the setup of OpenWhisk takes up to 8 minutes.  This setup was started when you started this scenario.
It is recommended that you come back in 8 minutes to proceed with the first step in this scenario.  We expect to fix
this 8 minute delay soon**